pkg/gadgets/run/tracer/tracer.go

```diff
@@ -356,6 +364,29 @@ func (t *Tracer) attachProgram(gadgetCtx gadgets.GadgetContext, p *ebpf.ProgramS
 			Name:    p.AttachTo,
 			Program: prog,
 		})
+	case ebpf.SchedCLS:
+		parts := strings.Split(p.SectionName, "/")
+		if len(parts) != 3 {
+			return nil, fmt.Errorf("invalid section name %q", p.SectionName)
+		}
+		if parts[0] != "classifier" {
+			return nil, fmt.Errorf("invalid section name %q", p.SectionName)
```

Comment:
In the documentation, we will need to say we don't support tc ebpf programs that uses tc's ELF convention. For examples, those programs won't work:
https://github.com/iproute2/iproute2/tree/main/examples/bpf

And I guess the documentation should say which ELF convention we support instead...